cdap-common/src/main/java/io/cdap/cdap/common/internal/remote/RemoteClient.java

```diff
@@ -124,7 +132,7 @@ public HttpResponse execute(HttpRequest request) throws IOException, Unauthorize
       }
       if (HttpCodes.isRetryable(responseCode)) {
         String message = String.format("Service %s is not available: %s",
-                                       discoverableServiceName, response.getResponseBodyAsString());
+                                       discoverableServiceName, getResponseBody(response));
```

Comment:
Why are we doing it here? It should be responsibility of the HTTP client

Reply:
Fixed common-http - https://github.com/cdapio/common/pull/59. Also updated this PR to use new common-http version